Bio/SeqIO/pepdigestIO.py

```diff
@@ -0,0 +1,143 @@
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+#
+# This module is for reading .pepdigest format files as SeqRecord objects.
+"""Bio.SeqIO support for the "pepdigest" file format.
+
+You are expected to use this module via the Bio.SeqIO functions.
```

Comment:
This needs to explain what "pepdigest" files are, and where they come from. Ideally include a URL.